python/mxnet/callback.py

```diff
@@ -37,19 +37,34 @@ def _callback(iter_no, sym=None, arg=None, aux=None):
 
 
 def do_checkpoint(prefix, period=1):
-    """Callback to checkpoint the model to prefix every epoch.
+    """A callback that saves a model checkpoint every few epochs. Checkpoint is binary files
```

Comment:
Thanks madan for catching this. 

A few changes to your changes: 
* "made up of *a* couple of ..."
* "and the parameters file is named ..."